### PR TITLE
fix(java_indexer): add flag to workaround JDK bug on Java 11

### PR DESCRIPTION
**Pull request by robinp-tw**
The --ignore_typesig_exceptions helps the indexer to continue even if hitting JDK-8212750.

Addresses an indexer exception mentioned in #4464.

TESTED=built using Bazel 2.1 using my `nix` branch and ran indexing. Tried to build using plain Bazel 3.0 on master, but hit some woes.

### CODE REVIEW

**Comment by shahms** [comment]
Robin would you mind signing the CLA? I think perhaps you had a different username before?

**Comment by robinp-tw** [comment]
Well spotted :) Will do!

**Comment by robinp-tw** [comment]
@googlebot I signed it!

File: kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java [hunk 1]
```diff
@@ -24,6 +24,15 @@
   @Parameter(names = "--emit_jvm_signatures", description = "Generate vnames with jvm signatures.")
   private boolean emitJvmSignatures;
 
+  @Parameter(
+      names = "--ignore_typesig_exceptions",
```
Comment (shahms) [inline kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: This seems broadly reasonable to me, but I might like a more descriptive flag name

Comment (robinp-tw) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: I'm open to suggestions.

Comment (schroederc) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: Do we need to flag this behavior?  It follows the standard Kythe doctrine of letting the indexers "do the best they can" even under error conditions.

Comment (shahms) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: That works for me, particularly with the error message indicating the workaround.

Comment (robinp-tw) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: Just to clarify: you propose to remove the flag? If so, will do.

Comment (shahms) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java:28]: Yeah, just remove the flag.

Comment (robinp-tw) [reply kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java]: Done.

**Comment by robinp-tw** [comment]
Ping - please merge if things are green.

**Comment by shahms** [comment]
Do you mind running google-java-format? I'm not sure the exact formatting complaint. (You can ignore the unrelated shell lint problem and "spelling error").

**Comment by shahms** [comment]
Adopting this with the formatting changes and closing this PR in favor of #4614

**Pull request closed without merging**